Library/Homebrew/github_packages.rb

```diff
@@ -17,6 +17,12 @@ class GitHubPackages
   DOCKER_PREFIX = "docker://#{URL_DOMAIN}/"
   URL_REGEX = %r{(?:#{Regexp.escape(URL_PREFIX)}|#{Regexp.escape(DOCKER_PREFIX)})([\w-]+)/([\w-]+)}.freeze
 
+  # Translate Homebrew tab.arch to OCI platform.architecture
+  TAB_ARCH_TO_PLATFORM_ARCHITECTURE = {
+    "arm64"  => "arm64",
+    "x86_64" => "amd64",
+  }.freeze
```

Comment:
I wonder if this should be in `hardware.rb` instead. I've seen this substitution pattern a few times in homebrew/core. (Not too common; I'll admit.)

Reply:
I think it's worth just keeping it scoped here for now given the specific use-case here.

Reply:
This particular vocabulary is specific to OCI and Go more generally, but yes, the problem of mapping one vocabulary to another comes up very often.